gem/templates/chhaajaa/base.html

```diff
@@ -218,7 +218,8 @@
                 <input type="checkbox" value="toggles" name="search_toggle" id="search_accordion" class="nav-search__checkbox" />
                 <div class="nav-search__wrapper">
                   <form action="{% url 'search' %}" class="nav-search__form" novalidate>
-                    <input name="q" type="search" aria-label="Search Input" placeholder="{% trans 'Search' %}">
+                    <label class="search__label" for="search_top">Search</label>
```

Comment:
Shouldn't "Search" here (and in the other templates) be translated?

Reply:
Now updated.